tests/unit/test_rbac_resolvers_key_value_pair.py

```diff
@@ -818,3 +818,86 @@ def test_admin_permissions_for_user_scoped_kvps(self):
             resource_db=kvp_1_db,
             permission_types=self.write_permission_types,
         )
+
+    def test_user_permissions_for_system_scope_kvps(self):
```

Comment:
There are already a number of tests on user permissions for system scope kvps rewritten under KeyValueSystemScopePermissionsResolverTestCase in my last commit.  What's added here that's different than those tests?

Reply:
Added the unit test as the list functionality was pending for user with system scope kvps. It got skipped from our mind that it was already written.
We will take care of this in future commits.